Training24/intlist.c

```diff
@@ -0,0 +1,108 @@
+// ------------------------------------------------------------------------------------------------
+// Training ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.
+// ------------------------------------------------------------------------------------------------
+// intlist.c
+// Program on A2 branch.
+// ------------------------------------------------------------------------------------------------
+
+#include <stdio.h>
+#include <malloc.h>
+#include "intlist.h"
+
+void Create (LinkList* list) {
+   list->head = NULL;

```

Comment:
You are not allocating any memory to list here, so the list itself does not exist. This code does essentially nothing.